packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -220,6 +220,7 @@ class SnapWPConfigManager {
 				if ( cfg[ key ] === undefined ) {
 					delete cfg[ key ];
 				} else if (
+					// This should probably be moved into the schema as a sanitize callback.
```

Comment:
It's the keyword that allows these to be searchable i.e. actionable.

(if we still havnt fixed the eslint syntax warning then `// TODO:` is a good replacement for `// @todo`

```suggestion
					// @todo This should probably be moved into the schema as a sanitize callback.
```

Reply:
Oh no, my bad. Forgot to add the keyword.